### PR TITLE
[14.0][FW] [13.0][IMP]base_user_role: Groups-roles navigation

### PR DESCRIPTION
**Pull request by sebalix**
Port of #157 from 13.0 to 14.0.

### CODE REVIEW

**Comment by GuillemCForgeFlow** [comment]
Hi @sebalix, I opened this FW: https://github.com/OCA/server-backend/pull/159 before I saw that this PR had been created. If you agree, you can close this PR as the one I opened already has some changes that were necessary on v14.0 and it has the approved label already. Let me know if that sounds good! :smile:

File: base_user_role/models/group.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        for group in self:
+            if group.trans_parent_ids:
+                group.role_ids = group.trans_parent_ids.role_id
+            else:
+                group.role_ids = group.role_id
+
+    def action_view_roles(self):
+        self.ensure_one()
+        action = self.env["ir.actions.act_window"].for_xml_id(
+            "base_user_role", "action_res_users_role_tree"
```
Comment (GuillemCForgeFlow) [inline base_user_role/models/group.py:64]:
```suggestion
        action = self.env["ir.actions.act_window"]._for_xml_id(
            "base_user_role.action_res_users_role_tree"
```
Here are the changes I mentioned

**Comment by GuillemCForgeFlow** [comment]
Also, I will be opening a FW for v15.0. @sebalix

**Comment by sebalix** [comment]
@GuillemCForgeFlow all good, I close this one, thanks!

**Pull request closed without merging**